### PR TITLE
feat: add standardising script (TDE-303)

### PR DESCRIPTION
**Pull request by paulfouquet**
## Description
Add the `standardising` script

## Changes

- `standardising.py` has been added in the `script` directory
- Unit tests have been initialized
- some changes in `aws_helper` related to assuming a role

### CODE REVIEW

File: scripts/aws_helper.py [hunk 1]
```diff
@@ -49,6 +52,10 @@ def get_credentials(bucket_name: str):
                 assumed_role_object["Credentials"]["SecretAccessKey"],
                 assumed_role_object["Credentials"]["SessionToken"],
             )
+        if set_credentials:
```
Comment (blacha) [inline scripts/aws_helper.py:55]: We should never overwrite the system environment variables.

You should take these credentials and add them into the environment of the command

File: scripts/standardising.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+src_bucket_name, src_file_path = aws_helper.parse_path(source)
+dst_bucket_name, dst_path = aws_helper.parse_path(destination)
+get_log().debug("source", bucket=src_bucket_name, file_path=src_file_path)
+get_log().debug("destination", bucket=dst_bucket_name, file_path=dst_path)
+dst_bucket = aws_helper.get_bucket(dst_bucket_name)
+
+with tempfile.TemporaryDirectory() as tmp_dir:
+    standardized_file_name = f"standardized_{os.path.basename(src_file_path)}"
+    tmp_file_path = os.path.join(tmp_dir, standardized_file_name)
+    src_gdal_path = f"/vsis3/{source.replace('s3://', '')}"
```
Comment (blacha) [inline scripts/standardising.py:28]: replce the `s3://` with `/vsis3` this will let it run on files that are not in s3.

Comment (paulfouquet) [reply scripts/standardising.py:28]: My bad, stupid copy/paste from `Topo-Processor`...

File: scripts/aws_helper.py (hunk 1 above)
Comment (paulfouquet) [reply scripts/aws_helper.py:55]: Yes, I was concerned by removing the default credentials by doing this.

**Review by amfage — APPROVED** [review]

**Pull request merged**